esmvaltool/cmorizers/data/downloaders/datasets/esacci_l4_ghrsst_gmpe.py

```diff
@@ -0,0 +1,39 @@
+"""Script to download cds-satellite-albedo from the CDS."""
```

Comment:
It is unclear to me which dataset this script is about because the script name does not match the header. Could you please check?

Reply:
I will take a look, but is unclear even for me: downloader, cmorizer and yaml files are pointing to three different sources.

And it looks like the downloader is not a copy paste error as the product name is unique in the project

Reply:
I still struggle to see where this is coming from. As you said the product name does not appear anywhere in the project. I'd delete this before merging if we can't find an explanation for this.